sylt-parser/src/statement.rs

```diff
@@ -168,16 +191,29 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
             }
         },
 
-        // `use a`
-        [T::Use, T::Identifier(name), ..] => (
-            ctx.skip(2),
-            Use {
-                file: Identifier {
-                    span: ctx.skip(1).span(),
-                    name: name.clone(),
-                },
-            },
-        ),
+        // `use path/to/file`
+        // `use path/to/file as alias`
+        [T::Use, ..] => {
+            let (path_ctx, path) = path(ctx.skip(1))?;
+            let (ctx, alias) = match &path_ctx.tokens[path_ctx.curr..] {
+                [T::As, T::Identifier(alias), ..] => (path_ctx.skip(2), Some(alias.clone())),
+                [T::As, ..] => return Err((
+                    skip_until!(path_ctx, T::Newline),
+                    vec![syntax_error!(path_ctx.skip(1), "Expected alias")]
+                )),
```

Comment:
Preferably use `raise_syntax_error`
```suggestion
                [T::As, ..] => raise_syntax_error(
                    skip_until!(path_ctx, T::Newline),
                    "Expected alias"
                ),
```
I haven't checked if this compiles.